views/pages/funding/programme-detail.njk

```diff
@@ -72,5 +72,11 @@
                 </div>
             {% endif %}
         </div>
+
+        {# Case Studies #}
+        {{ caseStudyCollection(
+            caseStudies = entry.caseStudies,
+            customSectionTitle = __('global.misc.projectExamples'
```

Comment:
The linebreaks here make this a bit weird, can you put the closing `)` on the same line?